docs: add public collections gallery howto

ikreymer opened the pull request:
Adding a 'howto' section on how to enable public collections gallery, extracted from blog post - to be linked from the blog post.

ikreymer committed:
add public collections gallery howto

ikreymer committed:
fix paths

Shrinks99 commented:
From @Shrinks99 in Discord chat:

> I think I'd rather surface this type of how to content in an [admonition](https://docs.browsertrix.com/develop/docs/#admonitions) closer to the information in the docs about what the guide is covering.  This isn't a particularly long how-to guide and we already have a page that is linked from the public collections docs that — in tandem with the in-app copy — should give a decent path for users to follow? 
We also have a section on sharing collections already: https://docs.browsertrix.com/user-guide/presentation-sharing/#sharing-collections so that part of the guide is duplicate content

ikreymer committed:
docs: update collection howto

ikreymer committed:
docs work

Shrinks99 committed:
Collections guide formatting fixes

- Whitespace fixes
- List item dashes (used elsehwere)

Shrinks99 committed:
Add correct icons

Shrinks99 committed:
Em dashes, minor wording changes

DaleLore committed:
Review: Collections docs edit (#2465)

Broke down more steps within Presentation & Sharing

Cleaned up some areas where the page looped back to Intro to Collections
and Enable Public Collections (now Intro to Public Collections)

I need help with some of the icons please 🙏
And need extra eyes for Copy

---------

Co-authored-by: Ilya Kreymer <ikreymer@gmail.com>
